### PR TITLE
✨ Add ReactRouter and Link to HomePage

### PR DESCRIPTION
**Pull request by Sina-Sche**
- [x]  Install React Router

- [x] Setup first Page (HomePage)

- [x] Basic Styling

Styling is not responsive yet, works best for MobileView(e.g. Iphone X)

### CODE REVIEW

**Comment by kschmitz123** [comment]
cool job 🧜🏼

File: client/src/components/ProductCard.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import HairbrushImg from "../assets/images/hairbrush.jpg";
 
 const Container = styled.div`
```
Comment (kschmitz123) [inline client/src/components/ProductCard.js:5]: your product container disappears behind the bottom nav ( just a little bit) . you might wanna add some padding there :)

Comment (Sina-Sche) [reply client/src/components/ProductCard.js:5]: ah,  true! Didn't pay attention to that! Thanks :)

**Review by Frenzy265 — APPROVED** [review]
lgtm! 👏🏻

**Review by Naasgul — APPROVED** [review]
good job! :)

**Pull request merged**